pyvista/themes.py

```diff
@@ -0,0 +1,1948 @@
+        --------
+        Create a custom theme from the default theme and load it into
+        pyvista.
+
+        >>> import pyvista
+        >>> from pyvista.themes import DefaultTheme
+        >>> my_theme = DefaultTheme()
+        >>> my_theme.font.size = 20
+        >>> my_theme.font.title_size = 40
+        >>> my_theme.cmap = 'jet'
+        ...
+        >>> pyvista.global_theme.load_theme(my_theme)
+        >>> pyvista.global_theme.font.size
+        20
+
+        Create a custom theme from the dark theme and load it into
+        pyvista.
+
+        >>> from pyvista.themes import DarkTheme
+        >>> my_theme = DarkTheme()
+        >>> my_theme.show_edges = True
+        >>> pyvista.global_theme.load_theme(my_theme)
+        >>> pyvista.global_theme.show_edges
+        True
+
+        """
+        if not isinstance(theme, DefaultTheme):
+            raise TypeError('``theme`` must be a pyvista theme like '
+                            '``pyvista.themes.DefaultTheme``')
+
+        for name, value in vars(theme).items():
+            setattr(self, name, value)
+
+    def _to_dict(self):
+        """Recursively convert to a dictionary for serialization."""
+        def to_dict(dict_):
+            dict_out = {}
+            for key, value in dict_.items():
+                if hasattr(value, '__dict__'):
+                    dict_out[key] = to_dict(value.__dict__)
+                else:
+                    dict_out[key] = value
+            return dict_out
+
+        return to_dict(self.__dict__)
+
+    def save(self, filename):
+        """Serialize this theme to a json file.
+
+        Examples
+        --------
+        Export and then load back in a theme.
+
+        >>> import pyvista
+        >>> theme = pyvista.themes.DefaultTheme()
+        >>> theme.background = 'white'
+        >>> theme.save('my_theme.json')  # doctest:+SKIP
+        >>> loaded_theme = pyvista.load_theme('my_theme.json')  # doctest:+SKIP
+        """
+        with open(filename, 'w') as f:
+            json.dump(self._to_dict(), f)
+
+
+class DarkTheme(DefaultTheme):
+    """Dark mode theme.
+
+    Black background, "viridis" colormap, tan meshes, white (hidden) edges.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> from pyvista import themes
+    >>> pyvista.set_plot_theme(themes.DarkTheme())
+
+    Alternatively, set via a string.
+
+    >>> pyvista.set_plot_theme('dark')
+
+    """
+
+    def __init__(self):
+        """Initialize the theme."""
+        super().__init__()
+        self._name = 'dark'
+        self._background = 'black'
+        self._cmap = 'viridis'
+        self._font.color = 'white'
+        self._show_edges = False
+        self._color = 'tan'
+        self._outline_color = 'white'
+        self._edge_color = 'white'
+        self._axes.x_color = 'tomato'
+        self._axes.y_color = 'seagreen'
+        self._axes.z_color = 'blue'
+
+
+class ParaViewTheme(DefaultTheme):
+    """Set the theme to a paraview-like theme.
```

Comment:
"Set the theme to a paraview-like theme." -> "A paraview-like theme."

Reply:
Hi @natter1,

Thanks for all the suggestions!  You can make comments and [contributions directly within GitHub](https://til.hashrocket.com/posts/szgdi4at59-github-insert-a-suggestion-#:~:text=Today%20I%20learned%20that%20GitHub,it%20with%20your%20proposed%20change.):

If you want to do multi-line suggestions:
https://github.blog/changelog/2020-02-26-multi-line-code-suggestions-beta/

It's super helpful, and you also get credit for any code suggestions (if you're into that sort of thing).

Reply:
Cool, was looking for that, but couldn't find it. ^^ Will change comments into suggestions